### ISSUE TITLE
Disable Video Card

### ISSUE THREAD

**Issue by rizedr** [Eun/DisableMonitor#65]
Any plans into updating this tool to disable the video card from which the Display Output?

Asking this because I am rocking a hackintosh build with 1x GTX 1080 as unsupported card & 1x GTX 750 Ti which works pretty well, and as a result my Graphic/Displays list looks like this:
![screen shot 2017-02-13 at 11 04 43](https://cloud.githubusercontent.com/assets/3323353/22898460/3f6c9b52-f1dc-11e6-9ff1-05be28ba9590.png)

**Comment by xtraa** [comment]
Jup same request for radeon rx480

**Comment by reestr** [comment]
Yes I would like this also.  gfcCardStatus won't work for egpu implementation and I want a way to prevent my integrated graphics being used by applications completely.

**Comment by Eun** [comment]
I have not found a way yet to disable the graphics card. It might be possbile with some drivers and tricky hooking code. However I think that could brick your entire OSX system. So wont do that...
